demux/basemask.py

```diff
@@ -1,62 +1,94 @@
 # -*- coding: utf-8 -*-
-
-import sys
 import click
 import logging
 
-from pathlib import Path
+from path import Path
 import xml.etree.cElementTree as et
```

Comment:
I know this is not your doing but `et` is very anonymous. Is there a more mnemonic name? xml_etree perhaps?